src/test/kotlin/uk/gov/dluhc/registercheckerapi/testsupport/testdata/models/RegisterCheckResultRequestBuilder.kt

```diff
@@ -65,6 +69,33 @@ fun buildRegisterCheckMatchRequest(
     registeredEndDate = registeredEndDate,
     applicationCreatedAt = applicationCreatedAt,
     franchiseCode = franchiseCode,
-    postalVote = null,
-    proxyVote = null
+    postalVote = postalVote,
+    proxyVote = proxyVote
+)
+
+fun buildPostalVote(
+    postalVoteUntilFurtherNotice: Boolean = true,
```

Comment:
The other builders use the full field name as the parameter name, which is why I did it here i.e. `postalVoteForSingleDate` rather than `forSingleDate` as the property of `PostalVote` is `postalVoteForSingleDate`. I agree it's really verbose, but for consistency I think we should leave it as it is. But I'm happy to change it if you think that's better?

Reply:
sorry clarification, meant to have `postalVoteUntilFurtherNotice = false` and `postalVoteForSingleDate` not null to match the structure of the default arguments for other voting arrangements builders

Reply:
Yes that makes sense